docs/Caching.md

```diff
@@ -0,0 +1,158 @@
+# Caching
+
+Web platform apps cache data at a few different levels in order to improve UI
```

Comment:
I'd go with `Data is cached at a few different levels…` since the platform isn't always responsible for caching